kubetools/deploy/commands/restart.py

```diff
@@ -54,8 +54,7 @@ def get_restart_objects(build, app_names=None, force=False):
     deployment_name_to_pods = defaultdict(list)
 
     for pod in pods:
-        if pod.metadata.owner_references:
-            if len(pod.metadata.owner_references) == 1:
+        if pod.metadata.owner_references and len(pod.metadata.owner_references) == 1:
                 owner = pod.metadata.owner_references[0]
```

Comment:
There's a double indent here now!

Reply:
That's what I get for committing without properly looking, if only github CI was smart enough to deal with it all.